src/utils/userStatus.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+import { t } from '@nextcloud/l10n'
+
+import type {
+	Conversation,
+	Participant,
+	ParticipantSearchResult,
+	ParticipantStatus,
+} from '../types'
```

Comment:
```suggestion
} from '../types/index.ts'
```

Reply:
😕 Either we:
- keep it as-is (Typescript works and doesn't complain for me)
- or adjust eslint and change in all occurences (~14 for now):
![image](https://github.com/user-attachments/assets/db666ae1-50bf-4d14-ab87-811256212d45)

Reply:
> * or adjust eslint

This message is not from ESLint, but from PhpStorm with Node.js (`require`) style import.

![image](https://github.com/user-attachments/assets/ebc94bb8-1410-44e3-92fc-7526262b192d)

Reply:
It works, because both Webpack and TS by default uses old import processing - like it is not ESM but CJS (`require` from Node.js). But in web and native Nodejs ESM, `imports` must have exact path. This is also why modern tools like Vite ask to use full path even there is a support to resolve extensions or index.